.github/workflows/build-and-deploy.yml

```diff
@@ -449,6 +449,23 @@ jobs:
           deployment_id: ${{ steps.deployment.outputs.deployment_id }}
           env_url: ${{ steps.deploy_review.outputs.deploy-url }}
 
+  merge-dependabot:
+    name: Merge dependabot
+    if: ${{ github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'dependencies') }}
+    needs: [lint, test, deploy-review-app]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Merge minor dependency updates
+        uses: fastify/github-action-merge-dependabot@v3.2.0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          target: minor
+          exclude: 'govuk-components,govuk_design_system_formbuilder,govuk-frontend'
```

Comment:
@cwrw great, thank you. 👍  We can keep this under review and always change our minds later.

We could add the `dfe-autocomplete` front-end npm package, but that's currently installed directly via GitHub instead of as an npm package, so it might not get picked up by dependabot anyway?

Reply:
yea reading up on docs it doesn't pick those up so should be good